app/views/allocations/_pom_tables.html.erb

```diff
@@ -30,13 +30,13 @@ Recommendation: <%= @recommended_pom %> POMs
       <td aria-label="Tier D cases" class="govuk-table__cell "><%= pom.tier_d %></td>
       <td aria-label="Total" class="govuk-table__cell "><%= pom.total_cases %></td>
       <td aria-label="Working pattern" class="govuk-table__cell"><%= pom.working_pattern %></td>
-      <% if current_page?(action: 'show') %>
-        <td aria-label="Action" class="govuk-table__cell table_cell__left_align"><a
-          <%= link_to "Allocate", new_allocations_path(@prisoner.offender_no, pom.staff_id), class:"govuk-link"%></a></td>
-      <% else %>
-        <td aria-label="Action" class="govuk-table__cell table_cell__right_align"><a
-          href="#">Reallocate</a></td>
-      <% end %>
+      <td aria-label="Action" class="govuk-table__cell table_cell__left_align">
+        <% if current_page?(action: :new) %>
```

Comment:
Currently allocating and re-allocating use the same controller methods so this if block does not currently work. Will fix in next PR